juniper_official/Server-Monitoring/kube-metrics/check-node-status.rule

```diff
@@ -0,0 +1,75 @@
+healthbot {
+    topic kube-metrics {
+        rule check-node-status {
+            keys [ condition node ];
+            synopsis "Node status KPI";
+            description "For each node, we track info about NetworkUnavailable, MemoryPressure, DiskPressure, PIDPressure, etc  and ensure that these are healthy";
+            sensor node-status {
+                description "Checks node status";
+                server-monitoring {
+                    sensor-name /kube/node;
+                    frequency 60s;
+                }
+            }
+            field condition {
+                sensor node-status {
+                    path condition;
+                }
+                type string;
+                description "Node condition";
+            }
+            field node {
+                sensor node-status {
+                    path node;
+                }
+                type string;
+                description "Node name";
+            }
+            field node_status {
+                constant {
+                    value "{{value}}";
+                }
+                type integer;
+                description "Field to check condition";
+            }
+            field status {
+                sensor node-status {
+                    path status;
+                }
+                type string;
+                description "Status of the node";
+            }
+            trigger node-status {
+                frequency 1offset;
+                term available {
+                    when {
+                        matches-with "$status" false {
+                            ignore-case;
+                            time-range 3offset;
+                            any;							
+                        }
+                    }
+                    then {
+                        status {
+                            color green;
+                            message "$condition for $node is $status";
+                        }
+                    }
+                }
+                term notavailable {
```

Comment:
Consider hyphenating for better readability.

Reply:
Done.